data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset

```diff
@@ -31,7 +26,7 @@ asset.export("layer", layer)
 
 asset.meta = {
   Name = "WAC [Utah]",
-  Version = "1.1",
+  Version = "1.0",
```

Comment:
This is not correct? (reducing the version number). Should it be  2.0?

Reply:
Indeed